src/components/Carousel.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Carousel = props => {
+  return typeof props.images === 'object' && props.images.length ? (
```

Comment:
Currently looking at implementing typescript for the project which will help with some of this